src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +38,237 @@ type State = AsyncComponent['state'] & {
-            {relays.map(
-              ({public_key, name, created, last_seen, first_seen, last_modified}) => (
-                <Content key={public_key}>
-                  <Info>
-                    <InfoItem>
-                      <Name>{name}</Name>
-                    </InfoItem>
-                    <InfoItem>
-                      <PublicKey>{public_key}</PublicKey>
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Created:')} date={created} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('First Seen:')} date={first_seen} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Last Seen:')} date={last_seen} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Last modified:')} date={last_modified} />
-                    </InfoItem>
-                  </Info>
-                  <Button
-                    title={t('Delete Rule')}
-                    label={t('Delete Rule')}
-                    size="small"
-                    icon={<IconDelete />}
-                  />
-                </Content>
-              )
-            )}
-          </PanelBody>
-          <PanelAction>
-            <Button href={RELAY_DOCS_LINK} target="_blank">
-              {t('Read the docs')}
+        <SettingsPageHeader
+          title={t('Relays')}
+          action={
+            <Button
+              priority="primary"
+              size="small"
+              icon={<IconAdd size="xs" isCircled />}
+              onClick={this.handleOpenAddDialog}
+            >
+              {t('New Relay Key')}
             </Button>
-            <Button onClick={this.handleAdd} priority="primary">
-              {t('Add Relay')}
-            </Button>
-          </PanelAction>
-        </Panel>
+          }
+        />
+        <TextBlock>
+          {tct(
+            `Relay is a relay service built by Sentry. You can run this on-premise for your SDKs or server to customize data scrubbing, buffering retries and more. You can generate relay keys for access. For more on how to set this up, read the [link:docs].`,
+            {
+              link: <ExternalLink href={RELAY_DOCS_LINK} />,
+            }
+          )}
+        </TextBlock>
+        <ClassNames>
+          {({css}) => (
+            <PanelTable
+              isEmpty={relays.length === 0}
+              emptyMessage={t('No relays keys have been added yet.')}
+              headers={[t('Display Name'), t('Relay Key'), t('Date Created'), '']}
+              className={css`
+                grid-template-columns: repeat(3, auto) max-content;
+                > * {
+                  padding: ${space(1)};
+                  @media (min-width: ${theme.breakpoints[0]}) {
+                    padding: ${space(1)} ${space(2)};
+                  }
+                }
+              `}
```

Comment:
This could be a styled component instead of inline css.

Reply:
This is not possible, because the component "PanelTable" is a func component and not a styled component. It accepts only a className.

As we discussed offline, I would love to use the css provided by emotion in the prop className. Example: className={css`...`}, but it doesn't work...it has another return type, etc.... so I decided to use ClassNames 😉